### ISSUE TITLE
Small documentation correction

### ISSUE THREAD

**Issue by CodeRhymesLife** [cmlibs/zinc#38]
Hi,

I'm brand new to zinc. While I was setting up I think I found a very small "bug" in the documentation. The documentation says to add the root to PATH, but I think root\bin should be added to PATH.

Here's a more detailed explanation:

![image](https://cloud.githubusercontent.com/assets/2764891/14338393/c168399c-fc2b-11e5-9f0c-0dfafb7c0086.png)
(http://opencmiss.org/downloads.html#/package/zinc/download)

When I installed on my x64 Window 10 Pro machine, although I selected 'Add OpenCMISS-Zinc to the system PATH for all users' during the installation wizard, updating the PATH failed. A small dialogue popped up to warn me. So I followed the instructions in the above image and added the root folder, 'c:\Program Files\OpenCMISS-Zinc', to my PATH. I thought this was odd, but did it anyway. I then installed PyZinc without a problem. Finally, when I ran an example file python couldn't find the opencmiss module. After a bit of fiddling I tried adding the \bin folder, 'c:\Program Files\OpenCMISS-Zinc\bin', to my PATH and things began working.

Long story short, if the above is correct I think you will want to update the text in the image to say something like "... add \bin to PATH ...".

**Comment by alan-wu** [comment]
Thanks for reporting this, you are probably right. I will take a look and update the instruction. Let me know if you need any help using our libraries.

**Comment by CodeRhymesLife** [comment]
@alan-wu Np. I'm happy to help. Thank you for creating this great piece of software.

I actually do need some help. What's the best way to ask? Is there a forum where I can post questions? Should I just post them here? Right now I'm just trying to get the Cmgui examples working.

**Comment by alan-wu** [comment]
Using Github is a great way to inform us of any assistance you require 
as all developers have now moved to use it. Andre told me about the 
problem you have experienced with Cmgui and he should reply you soon 
with a solution.

You can also find the Cmgui Github page here: https://github.com/cmiss/cmgui

Cmgui examples testing display on the page is not relevant anymore 
however the latest developer version should work with most of the example.

Sorry about the inconvenience, we are in the middle of restructing our 
build systems but things will look better soon.

**Comment by CodeRhymesLife** [comment]
No problem, and thank you for the guidance. The developer version was the solution, now I'm up and running. I'm sure I will have more questions in the future, so thank you in advance.

**Comment by rchristie** [comment]
The errant text has been fixed to say 'bin' not 'top' folder.
